Formula/youtube-dlc.rb

```diff
@@ -0,0 +1,37 @@
+class YoutubeDlc < Formula
+  desc "Media downloader supporting various sites such as youtube"
+  homepage "https://github.com/blackjack4494/yt-dlc"
+  url "https://github.com/blackjack4494/yt-dlc/archive/2020.10.31.tar.gz"
+  sha256 "0d23cc2e2f1cc7291e5d2a468b1cd282a6be228a215df188aaecae8951ac64d9"
+  license "Unlicense"
+
+  livecheck do
+    url "https://github.com/blackjack4494/yt-dlc/releases/latest"
+    regex(%r{href=.*?/tag/v?(\d+(?:\.\d+)+)["' >]}i)
+  end
+
+  head do
+    url "https://github.com/blackjack4494/yt-dlc.git"
+  end
```

Comment:
This isn't needed and should just be 
```suggestion
  head "https://github.com/blackjack4494/yt-dlc.git"
```